docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -18,13 +18,15 @@ materialDesign: https://m2.material.io/components/date-pickers
 ## Component composition
 
 The component is built using the `DateField` for the keyboard editing and the `DateCalendar` for the view editing.
-All the documented props of those two components can also be passed to the Date Picker component.
 
 Check-out their documentation page for more information:
 
 - [Date Field](/x/react-date-pickers/date-field/)
 - [Date Calendar](/x/react-date-pickers/date-calendar/)
 
+You can check the available props of the combined component on the dedicated [API page](/x/api/date-pickers/date-picker/#props).
+Some [`DateField` props](/x/api/date-pickers/date-field/#props) are not available on the Picker component, you can use `slotProps.field` to pass them.
```

Comment:
Maybe

```suggestion
Some of the [`DateField` props](/x/api/date-pickers/date-field/#props) are not available on the Picker component, you can use `slotProps.field` to pass them to the field.
```

Reply:
Was this suggestion provided by your IDE spell-checker?
I do agree with the addition of the `to the field`. 👌 
Regarding the addition of the `of the` after `Some`, I'm not sure it's worth it and clarifies anything. 🤔 
It would also cause the paragraph to increase by one line on the `date-picker`, `time-picker`, and `date-time-picker` pages. 🙈

Reply:
> Was this suggestion provided by your IDE spell-checker?

No, purely me here.
I know that "Some XXXs" is valid "Some cities are next to the sea" for instance.
But in your sentence, it feels weird to me, not sure why.

Since I'm not a native english speaker I may be totally wrong :laughing:

Reply:
We can ask for @samuelsycamore input on the most proper approach here. 🙈 👌

Reply:
Thank you, updated with the suggested solution. 😉 🙏